se2_planning/include/se2_planning/OmplReedsSheppPlanner.hpp

```diff
@@ -56,6 +56,12 @@ class OmplReedsSheppPlanner final : public OmplPlanner {
   void setParameters(const OmplReedsSheppPlannerParameters& parameters);
   void setStateValidator(std::unique_ptr<StateValidator> stateValidator);
   const StateValidator& getStateValidator() const;
+  void lockStateValidator();
+  void unlockStateValidator();
+  bool isLocked() const;
+  void updateStateSpaceBounds(const ompl::base::RealVectorBounds& bounds) override;
```

Comment:
updateStateSpaceBounds can be moved to the OmplPlanner.hpp since it is ompl specific.

Reply:
Done in db4a618de5f69d9b9b0bee5375f1217ca36002a6.